GitUI/Resources/ChangeLog.md

```diff
@@ -1,6 +1,31 @@
 Changelog
 =========
 
+### [Version 2.51.01] (11 Mar 2018)
+
+#### Features:
+* TaskCanceledException for ShowSelectedFileDiff() - Issue [4570]
+* Add a hotkey for toggling viewing of Tags - Issue [4449]
+* Fixes on jira commit hint plugin - PR [4412]
+* Add two ScriptEvents, BeforeMerge, AfterMerge - Issue [4380]
+* Better display of "Find and replace" dialog on multiple screens computer - PR [4354]
+* Commit "Author" text box has incorrect descriptive text - Issue [3501]
+* Keep last used Destination in clone dialog as entered - Issue [2313]
+
+#### Fixes:
+* Find in the Diff tab shows differences reversed after wrapping around the files - Issue [4546]
+* "Restore to selected version" function is removed from 2.51 - Issue [4535]
+* PuTTY Installer Is Outdated - Issue [4509]
+* 2.51: App crashes upon pressing "Script"  - Issue [4488]
+* Find in the Diff tab fails to find text - Issue [4485]
+* Remotes not persisted - Issue [4441]
+* git log is called for artificial commits - Issue [4361]
+* "Reword Commit" doesn't refresh the UI - Issue [3935]
+* Submodule dialog: "Name" textbox too small for long names - Issue [1858]
+
```

Comment:
Should mention bundled git version

Reply:
Noted
Speaking of which, is it worth bundling 2.16.2 instead of 2.16.1.2?

Reply:
2.16.2.1 is the version to use and it is included in 2.51 (but not master)

Reply:
I can't see it here - https://github.com/git-for-windows/git/releases

Reply:
Filenames does not consistently include the 4th digit, but for instance for 2.16.1(4) the filenames includes 2.16.1.4 or 2.16.1.windows.4
DownloadExternals.cmd probably needs a tweak to handle the .1 releases

Reply:
I've bundled the latest 2.16.2 downloaded from the Git's release page